apps/ensindexer/src/lib/lib-config.ts

```diff
@@ -59,7 +59,7 @@ export function prettyPrintConfig(config: ENSIndexerConfig) {
         ]),
       ),
     } as ENSIndexerConfig,
-    null,
+    (key: string, value: unknown) => (key === "abi" ? `(truncated ABI output)` : value),
```

Comment:
Quite handy to avoid printing very long ABIs into stdout for ENSIndexer container.